content/course-section/calendar.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Calendar"
```

Comment:
Not sure, but we might want to have a "type" in addition to title. In 95% of cases, the title and the type should be the same, but it would give authors a little more flexibility to give a section a different label in the nav, but still identify it as a particular type of content.